x/cvm/client/cli/query.go

```diff
@@ -67,6 +78,8 @@ func GetCmdView() *cobra.Command {
 				if err != nil {
 					return err
 				}
+			} else {
```

Comment:
How about utilizing this method instead? https://pkg.go.dev/github.com/spf13/cobra#MarkFlagRequired

Reply:
I had added MarkFlagRequired check like this but it still processed the code so I added the else statement with example 
```go
_ = cmd.MarkFlagRequired(FlagCaller)
```

Reply:
Doesn't adding one line as follows work?
```
cmd.Flags().String(FlagCaller, "", "optional caller parameter to run the view function with")
cmd.MarkFlagRequired(FlagCaller)
```

Reply:
Why are we making it required?